cmd/cosign/cli/verify/verify_attestation.go

```diff
@@ -275,9 +275,10 @@ func (c *VerifyAttestationCommand) Exec(ctx context.Context, images []string) (e
 
 		// If a trusted root path is provided, we will use it to verify the bundle.
 		// Otherwise, the verifier will default to the public good instance.
-		co.TrustedMaterial, err = root.NewTrustedRootFromPath(c.TrustedRootPath)
-		if err != nil {
-			return fmt.Errorf("creating trusted root from path: %w", err)
+		// co.TrustedMaterial is already loaded from c.TrustedRootPath above,
+		// the check below is to serve as a reminder to developers and to future-proof.
+		if co.TrustedMaterial == nil {
+			panic("internal cosign error co.TrustedMaterial is nil with non-empty c.TrustedRootPath")
```

Comment:
If we don't delete the switch case then we could still just delete this, this check is done earlier in the file https://github.com/sigstore/cosign/blob/c36801ef34aa24eb01d064ab12da66a83b9c3ff4/cmd/cosign/cli/verify/verify_attestation.go#L153-L157

Reply:
I meant to leave it to guard against future changes, as mentioned in the comment
```go
// the check below is to serve as a reminder to developers and to future-proof.
```